index.js

```diff
@@ -324,10 +324,14 @@ function showNotification(notificationObj) {
           })
         }
 
+
+        // Display time per notification basis.
+        var displayTime = (notificationObj.displayTime ? notificationObj.displayTime : config.displayTime);
```

Comment:
Both work just fine.. there's nothing complicated going on so the placement does not matter.

Reason why (result ? "test" : "test2") is a good idea is to show that you are not using a function and in this case the function could be the "result" variable. If you would do it like: (result) ? "test" : "test2" it could be mistaken for a function etc. And ofc a function needs brackets as well but I hope you get the point.